rosboard/html/js/viewers/meta/Viewer.js

```diff
@@ -131,7 +131,7 @@ class Viewer {
     }
 
     if(data._warn) {
-      this.warn(data._warn);
+      // this.warn(data._warn);
```

Comment:
Maybe add a console debug message that the warning was suppressed.  Otherwise, just remove the condition check as well.

Reply:
> Maybe add a console debug message that the warning was suppressed. Otherwise, just remove the condition check as well.

Done, added a console debug message, it might be nice to know in the future if rosboard ever starts to slow down with large topics @jwdinius